### ISSUE TITLE
Embed used build tags into version output 

### ISSUE THREAD

**Issue by jacksgt** [go-gitea/gitea#828]
Hi,

I think it would be nice to be able to view the [tags / flags](https://docs.gitea.io/en-us/install-from-source/) which where used for building this particular Gitea binary.

Nginx has something like this:
```
$ nginx -V
nginx version: nginx/1.9.10
built with OpenSSL 1.0.2j  26 Sep 2016
TLS SNI support enabled
configure arguments: --with-cc-opt='-g -O2 -fPIE -fstack-protector-strong -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2' --with-ld-opt='-fPIE -pie -Wl,-z,relro -Wl,-z,now' --prefix=/usr/share/nginx --conf-path=/etc/nginx/nginx.conf --http-log-path=/var/log/nginx/access.log --error-log-path=/var/log/nginx/error.log --lock-path=/var/lock/nginx.lock --pid-path=/run/nginx.pid --http-client-body-temp-path=/var/lib/nginx/body --http-fastcgi-temp-path=/var/lib/nginx/fastcgi --http-proxy-temp-path=/var/lib/nginx/proxy --http-scgi-temp-path=/var/lib/nginx/scgi --http-uwsgi-temp-path=/var/lib/nginx/uwsgi --with-debug --with-pcre-jit --with-ipv6 --with-http_ssl_module --with-http_stub_status_module --with-http_realip_module --with-http_auth_request_module --with-http_addition_module --with-http_dav_module --with-http_geoip_module --with-http_gunzip_module --with-http_gzip_static_module --with-http_image_filter_module --with-http_v2_module --with-http_sub_module --with-http_xslt_module --with-stream --with-stream_ssl_module --with-mail --with-mail_ssl_module --with-threads --add-module=/build/nginx-5VKL9Q/nginx-1.9.10/debian/modules/nginx-auth-pam --add-module=/build/nginx-5VKL9Q/nginx-1.9.10/debian/modules/nginx-dav-ext-module --add-module=/build/nginx-5VKL9Q/nginx-1.9.10/debian/modules/nginx-echo --add-module=/build/nginx-5VKL9Q/nginx-1.9.10/debian/modules/nginx-upstream-fair --add-module=/build/nginx-5VKL9Q/nginx-1.9.10/debian/modules/ngx_http_substitutions_filter_module
```

Maybe for Gitea something like this could be implemented:
```
$ gitea -v 
Gitea version 1.0.1 built with: 'bindata' 'pam' 'sqlite'
```

**Comment by appleboy** [comment]
```
$ ./gitea -v
Gitea version 1.0.0+309-g6f76b49
```

**Comment by lunny** [comment]
resolved by #1051

**Comment by jacksgt** [comment]
Great work!
